Dockerfile

```diff
@@ -0,0 +1,22 @@
+# UI build
+FROM node:14-buster AS ui
+RUN apt-get update -y && apt-get install -y build-essential
+RUN mkdir -p /home/app/node_modules && chown -R node:node /home/app
+WORKDIR /home/app
+USER node
+COPY --chown=node:node package*.json /home/app/
+RUN npm install
+COPY --chown=node:node . /home/app/
+RUN make ui
+
+# Go build
+FROM golang:1.16 AS go-build
+COPY . /app
+COPY --from=ui /home/app/cmd/wego/ui/run/dist/ /app/cmd/wego/ui/run/dist/
+WORKDIR /app
+RUN make dependencies && make bin
+
+# Distroless
+FROM gcr.io/distroless/base
+COPY --from=go-build /app/bin/wego /wego
+ENTRYPOINT ["/wego"]
```

Comment:
I'm seeing that the docker image has ENTRYPOINT ["/wego"] not sure how we are going to start the api 🤔 Is there a way to tell the deployment to replace that entrypoint ? 🙂

Reply:
in a k8s deployment you set `args` to `["ui", "run"]`, or running locally 

`docker run -it --rm ghcr.io/weaveworks/wego-app:latest ui run`